src/main/resources/sql/init_db.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE shops (
+  id                  SERIAL NOT NULL CONSTRAINT shop_key PRIMARY KEY,
+  name                VARCHAR(20),
+  phone_number        VARCHAR(20),
+  --   type
```

Comment:
I think it is supposed to be a type of product? if so you can user a regular varchar. it is not necessary to create a separate sql type for it